### PR TITLE
Fix exports

### PR DESCRIPTION
**Pull request by quintaartsen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,99 @@
...
+```
+
+*Peer connecting to the network, it is possible to await the init to ensure a connection is made.*
+```ts
+class MainService {
+	private peer: IPeer;
+
+	constructor() {
+		this.peer = new Peer("typeOfPeer");
+		await this.peer.init([145.93.101.81]);
```
Comment (Svenmarim) [inline README.md]: Remove this IP

File: README.md [hunk 2]
```diff
@@ -1 +1,99 @@
...
+```
+
+  ### Usage:
+ *See the TSDoc of IPeer for more specific information*
+
+ ```ts
+// Create the peer
+const peer: IPeer = new Peer("examplePeer");
+// Connect to the p2p network and await the connection
+await peer.init("8081", ["145.93.101.81"]);
```
Comment (Svenmarim) [inline README.md]: and this IP

File: README.md [hunk 3]
```diff
@@ -1 +1,99 @@
...
+peer.registerReceiveHandlerForMessageType("testMessageTypeWithResponse", async (message: Message, senderGuid: string, response: RESPONSE_TYPE) => {
+    if (message && senderGuid) {
+        // Rspond to the message
+        response(message);
+    }
+});
+
+// Get a validator peer
+const validatorGuid: string | undefined = peer.getPeerOfType("Validator");
+if (validatorGuid !== undefined) {
```
Comment (Svenmarim) [inline README.md]: This can be replaced by `if (!validatorGuid)`, right???

**Review by Svenmarim — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,99 @@
...
+
+	constructor() {
+		this.peer = new Peer("typeOfPeer");
+		await this.peer.init(["0.0.0.0"]);
+	}
+}
+```
+
+  ### Usage:
+ *See the TSDoc of IPeer for more specific information*
```
Comment (hugovandevliert) [inline README.md]: Maybe add the location of the IPeer class

File: src/interfaces/peer.ts [hunk 5]
```diff
@@ -43,6 +43,21 @@ export interface IPeer {
      */
     sendBroadcastAsync(message: Message, responseImplementation?: RESPONSE_TYPE): Promise<void[]>;
 
+
```
Comment (hugovandevliert) [inline src/interfaces/peer.ts]: Please remove this duplicate whiteline

**Review by hugovandevliert — CHANGES_REQUESTED** [review]

**Pull request merged**